### ISSUE TITLE
Automatic "Advanced Search" Form

### ISSUE THREAD

**Issue by jqnatividad** [ckan/ideas-and-roadmap#228]
Even if advanced search queries are documented in CKAN (#227), casual users will be hard pressed to remember Solr query syntax.

It'd be great if CKAN can automatically create Advanced Search forms.

A lot of work has already been done by @TkTech on this

https://github.com/ckan/ckanext-scheming/issues/178 

Most especially once scheming is merged into core cc @wardi
